docs/Project-Configuration/Stepback-Bisection.md

```diff
@@ -0,0 +1,38 @@
+# Stepback Bisection
+Evergreen's stepback bisection performs stepback by continuously reducing the amount of commits needed to test by half. To enable it, go to the project settings 'general' tab.
+
+## Motivation
+Traditionally, Evergreen performed linear stepback only. This is tasking each previous commit one by one. If stepback bisection is enabled, Evergreen will use bisection instead of linear.
```

Comment:
Rather than "tasking", I'd say something like "activates each previous inactive commit"

Reply:
Not sure why I put tasking oops....

Reply:
This thread may have been missed